sdks/nodejs/src/agonesSDK.js

```diff
@@ -19,7 +19,26 @@ const services = require('../lib/sdk_grpc_pb');
 
 class AgonesSDK {
 	constructor() {
-		this.client = new services.SDKClient('localhost:59357', grpc.credentials.createInsecure());
+		function port() {
+			const defaultPort = 59357;
+			const port = process.env.AGONES_SDK_GRPC_PORT;
+			if (port === undefined) {
+				console.log("Environment variable AGONES_SDK_GRPC_PORT not defined, using default port ${defaultPort}");
+				return defaultPort;
+			}
+			const portNum = parseInt(port, 10);
+			if (isNaN(portNum)) {
+				console.log("Unable to parse ${port} defined in AGONES_SDK_GRPC_PORT into an integer\n");
```

Comment:
Nit: No \n required in console.log as it already includes a newline

Reply:
Thanks for catching this (I'd fixed it in one of the lines but not the other two).